modules/tinymce/tools/docs/tinymce.js

```diff
@@ -4,14 +4,15 @@
  * @static
  * @class tinymce
  * @borrow-members tinymce.EditorManager
+ * @borrow-members tinymce.EditorObservable
  * @borrow-members tinymce.util.Tools
  */
 
 /**
  * @property {tinymce.dom.DOMUtils} DOM Global DOM instance.
  * @property {tinymce.dom.ScriptLoader} ScriptLoader Global ScriptLoader instance.
- * @property {tinymce.AddOnManager} PluginManager Global PluginManager instance.
- * @property {tinymce.AddOnManager} ThemeManager Global ThemeManager instance.
+ * @property {tinymce.AddOnManager} PluginManager Global PluginManager instance. Subclass of AddOnManager.
```

Comment:
Oh, so we have the properties here. That's possibly all we need then. Also, just to make sure it's not missed this isn't a subclass, it's an instance of AddOnManager.